bundle-workflow/src/test_workflow/integ_test/local_test_cluster.py

```diff
@@ -22,12 +22,17 @@ class LocalTestCluster(TestCluster):
     Represents an on-box test cluster. This class downloads a bundle (from a BundleManifest) and runs it as a background process.
     """
 
-    def __init__(self, work_dir, bundle_manifest, security_enabled, s3_bucket_name):
+    def __init__(self, work_dir, component_name, additional_cluster_config, bundle_manifest, security_enabled,
+                 s3_bucket_name=None):
         self.manifest = bundle_manifest
         self.work_dir = os.path.join(work_dir, "local-test-cluster")
         os.makedirs(self.work_dir, exist_ok=True)
+        # Required by IM plugin as an additional config
+        os.makedirs("/tmp/tmp_dir", exist_ok=True)
```

Comment:
This will fail in many ways. What does the IM plug-in actually need?

Reply:
They need a valid location [here](https://github.com/opensearch-project/opensearch-build/pull/538/files#diff-b6591330ab14f0b34306021b8744f1e9c93e87a43e90cabee0024d9acb95a61aR25) for one of their integration test

Reply:
They write some stuff into that directory. @bowenlan-amzn might be able to answer in detail. I have opened an issue: https://github.com/opensearch-project/opensearch-build/issues/545

Reply:
Please open an issue in index-management to remove the need for this.

Reply:
Created the [issue](https://github.com/opensearch-project/index-management/issues/151) with IM team